google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java

```diff
@@ -155,30 +155,28 @@ private TranslateOptions(Builder builder) {
       apiKey = null;
       if (builder.apiKey != null) {
         logger.log(
-            Level.WARNING, "Ignoring API key: using explicit setting for credentials instead.");
+            Level.WARNING,
+                "Ignoring API key: using explicit setting for credentials instead.");
       } else if (getDefaultApiKey() != null) {
         logger.log(
             Level.WARNING,
-            String.format(
-                "Ignoring API key set in environment variable %s: using explicit setting for credentials instead.",
-                API_KEY_ENV_NAME));
+                "Ignoring API key set in environment variable {}: using explicit setting for credentials instead.",
+                API_KEY_ENV_NAME);
       }
     } else if (builder.apiKey != null) {
       credentials = null;
       apiKey = builder.apiKey;
       logger.log(
           Level.WARNING,
-          String.format(
-              "Ignoring Application Default Credentials: using explicit setting for API key instead.",
```

Comment:
String.format was missing %s earlier for arg  ServiceOptions.CREDENTIAL_ENV_NAME.